### ISSUE TITLE
datetime does not working in IE11

### ISSUE THREAD

**Issue by ShurikEv** [RobinHerbots/Inputmask#2238]
I use code:
```
        $('.js-schedule-edit').inputmask(
            "datetime", {
                inputFormat: "HH:MM",
                placeholder: "hh:mm",
                insertMode: true, 
                showMaskOnHover: false,
                hourFormat: 24
            }
        );
```

In Chrome it works fine. But in IE11 I have error:
> SCRIPT438: Object doesn't support property or method 'endsWith'
> jquery.inputmask.js (1898,45)

Example: https://jsfiddle.net/y3wnL19s/
- OS: Win10
- Browser: IE11
- Inputmask version: 5.0.0
![error](https://user-images.githubusercontent.com/427566/70132992-dc5b0400-16b7-11ea-8415-1f00a73ec820.png)

**Comment by RobinHerbots** [comment]
@ShurikEv , 

I will fix this by a regex instead of the endwidth.

**Comment by RobinHerbots** [comment]
@ShurikEv, 

This is fixed in version 5.0.1-beta.0
